appsec-kit-backend/src/main/java/com/vaadin/appsec/backend/VaadinVersion.java

```diff
@@ -0,0 +1,22 @@
+/*-
+ * Copyright (C) 2023 Vaadin Ltd
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ *
+ * See <https://vaadin.com/commercial-license-and-service-terms> for the full license.
+ */
+package com.vaadin.appsec.backend;
+
+import java.util.Objects;
+
+/**
+ * Contains the supported Vaadin versions.
+ */
+public enum VaadinVersion {
+    V7, V8, V24;
+
+    public static boolean isFlow(VaadinVersion version) {
+        return Objects.requireNonNull(version) == VaadinVersion.V24;
+    }
+}
```

Comment:
Can't we just infer this information from the Maven SBOM? If there's Flow, we look for NPM deps otherwise not.

Reply:
Good point. Yes, we can scan the SBOM for example for `flow-server` dependency. I removed the `VaadinVersion` and added components scan to decide if Flow is used.